fmriprep/interfaces/reports.py

```diff
@@ -169,10 +169,9 @@ def _generate_segment(self):
         if counts:
             header = '\t\t<ul class="elem-desc">'
             footer = '\t\t</ul>'
+            s = '' if n_runs == 1 else 's'
             lines = [
-                '\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'.format(
-                    task_id=task_id, n_runs=n_runs, s='' if n_runs == 1 else 's'
-                )
+                f'\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'
```

Comment:
A more terse version of `"s" if n_runs > 1 else ""`:

```suggestion
                f'\t\t\t<li>Task: {task_id} ({n_runs:d} run{"s"[:n_runs > 1]})</li>'
```

Reply:
😀 Nice. But perhaps too terse?

Reply:
It still fails. I'll probably revert.